### DISCUSSION TITLE
User782Tec 的更新频道

### DISCUSSION THREAD

**Discussion by User782Tec** [win12-online/win12#75] [Show and tell]
User782Tec 的更新频道
支持一下呗qwq（
所有更新均会加入至windows12正式版

**Comment by User782Tec** [comment]
VSCode
![vscode](https://user-images.githubusercontent.com/115236204/234136854-6f91ad73-4465-4a5f-b5dc-695615df47f2.png)

**Comment by User782Tec** [comment]
哔哩哔哩
![bilibili](https://user-images.githubusercontent.com/115236204/234136915-6018c082-eb3f-4a61-bc7a-347802cd05b5.png)

---

v5.1.1已更新

**Comment by User782Tec** [comment]
修复bug
修复前：
![image](https://user-images.githubusercontent.com/115236204/234255654-1f8c8e67-02ac-4b5f-a9ae-e155cfed1bc5.png)
修复后：
![image](https://user-images.githubusercontent.com/115236204/234255787-d2cc84e9-2705-4f4a-8495-407e424ca3a6.png)

**Comment by User782Tec** [comment]
计算器优化
优化前：
![image](https://user-images.githubusercontent.com/115236204/234574032-944428a8-cca9-4f26-a6dc-6f149423bb4c.png)
优化后：
![image](https://user-images.githubusercontent.com/115236204/234573781-bb8ebc8c-f0b9-42fc-b152-8d159bf3e9e0.png)

当输入框内只有0时，点击数字后可自动覆盖0
清除时会将输入框内容设为0
当输入框内只有一位数字时，退格会将其变为0
小组件已同步更改

**Reply by tjy-gitnub** [reply]
wow
所以是怎么做到的（更了吗？

**Reply by User782Tec** [reply]
> wow 所以是怎么做到的（更了吗？

没更呢

---

就是计算时替换字符

**Comment by User782Tec** [comment]
优化
关闭焦点窗口后，焦点会转移至下面一个窗口
更加符合原生体验

**Comment by tjy-gitnub** [comment]
为什么你要把webapp的内容放js里啊
?u?)o

**Reply by tjy-gitnub** [reply]
![image](https://user-images.githubusercontent.com/71509955/235350703-e9be5960-cc32-4c85-b496-5096cdb9c206.png)

**Reply by User782Tec** [reply]
> 为什么你要把webapp的内容放js里啊 ?u?)o

It will raise an error if I don't add them into the file.

**Reply by tjy-gitnub** [reply]
it's unblievable! what kind of error will be raised? it's just a iframe.

**Reply by User782Tec** [reply]
> it's unblievable! what kind of error will be raised? it's just a iframe.

![image](https://user-images.githubusercontent.com/115236204/235436901-dc5eaf90-c2f0-44ec-98bb-d270d7b5fdc8.png)

If I don't define the function "init()", it will raise TypeError.
And I let the iframes load after the "onload" event of the window so that the iframes may not block the page from loading resources. By Doing that, the startup (Page loading) time will shrink a lot than I don't do that. (自己写的 语法可能不对 抱歉

**Comment by User782Tec** [comment]
相机应用已更新

---

v5.2.0 已更新

**Comment by User782Tec** [comment]
![image](https://user-images.githubusercontent.com/115236204/235357961-77e7e443-9431-44d4-a0c5-653ad30dfa47.png)

登录界面
开只因音效

**Comment by User782Tec** [comment]
![image](https://user-images.githubusercontent.com/115236204/235358114-02889926-90c7-4005-af1d-e5c17190982c.png)

增加Favicon和标题

**Comment by User782Tec** [comment]
![image](https://user-images.githubusercontent.com/115236204/235358212-f8843239-fccf-4587-8f79-8c27814f7bac.png)

Edge浏览器细节优化

**Comment by User782Tec** [comment]
https://user-images.githubusercontent.com/115236204/235383856-012fde15-d7bc-493a-b98d-bcacf6cddcdd.mp4

登录界面
开机音效

---

v5.2.1 已更新

**Comment by User782Tec** [comment]
![image](https://user-images.githubusercontent.com/115236204/235444930-6c547afe-15da-40b2-829d-3c8cd3edcaab.png)

新登录界面

**Reply by tjy-gitnub** [reply]
哇哇哇哇哇哇哇哇哇哇哇哇
bg哪里找的！好好看

**Reply by tjy-gitnub** [reply]
另：是jpg吗，文件大吗😊
太大的话很影响体验啊

**Reply by User782Tec** [reply]
> 另：是jpg吗，文件大吗
> 太大的话很影响体验啊

是的
1.9MB，不算很大
这个分辨率挺高的，5120x2880，可以再压一压

**Reply by User782Tec** [reply]
> 哇哇哇哇哇哇哇哇哇哇哇哇 bg哪里找的！好好看

~~获取该壁纸的方法：~~
~~1. 制作最新版Kubuntu系统的安装盘（建议22.04.2LTS版本），并备份电脑数据~~
~~2. 在你的电脑上用该安装盘安装Kubuntu系统~~
~~3. 完成相关配置~~
~~4. 打开`/usr/share/wallpapers/Shell/contents/images/`目录~~
~~5. 找到`5120x2880.jpg`文件，保存在U盘上~~
~~6. 制作Windows 10系统的安装盘~~
~~7. 在你的电脑上用该安装盘安装Windows 10系统~~
~~8. 完成相关配置~~
~~9. 插入保存图片的U盘~~
~~10. 将U盘上的图片复制到目标文件夹~~
~~11. 双击查看图片，完成~~

发给你吧
![login](https://user-images.githubusercontent.com/115236204/235452936-a571b3b0-709b-4203-9c18-063b057c82c1.jpg)

**Comment by User782Tec** [comment]
我在没有缩小原图像大小的情况下
把登录界面的壁纸从1.9 MB压缩到了782.4 KB `(^V^)`
压缩后的壁纸：
![login](https://user-images.githubusercontent.com/115236204/235647492-97407ef3-3075-48ab-9022-2a0ceb3ba901.jpg)

**Comment by User782Tec** [comment]
![image](https://user-images.githubusercontent.com/115236204/236664747-3dac1806-f81d-4f72-a06a-66edeed3cb93.png)

可通过终端打开应用

**Reply by NB-Group** [reply]
> ![image](https://user-images.githubusercontent.com/115236204/236664747-3dac1806-f81d-4f72-a06a-66edeed3cb93.png)
> 
> 可通过终端打开应用

所以说输出不应该重定向到控制台吗 ( 我来增加工作量了嗨害嗨
![image](https://user-images.githubusercontent.com/121747915/236837854-3ca43e87-e6df-46ce-9d7d-a78438d93ff5.png)

**Comment by User782Tec** [comment]
![image](https://user-images.githubusercontent.com/115236204/236664963-ed4a9620-1244-4a4b-8dcd-3f1e494f3370.png)

Python解释器
支持多种功能
（感谢NB-Group提供的灵感）

**Reply by tjy-gitnub** [reply]
so...?
![image](https://github.com/tjy-gitnub/win12/assets/71509955/78b069af-30c7-4b12-8222-8cb2bf23ffaa)
可以加一个<input>吗？

**Comment by User782Tec** [comment]
![image](https://github.com/tjy-gitnub/win12/assets/115236204/ce0efe97-fb57-4b13-b35d-c81e88724f55)

运行应用

**Reply by NB-Group** [reply]
> ![image](https://user-images.githubusercontent.com/115236204/239657204-ce0efe97-fb57-4b13-b35d-c81e88724f55.png)
> 
> 运行应用

下拉文本框呢？
![image](https://github.com/tjy-gitnub/win12/assets/121747915/8ff0f02d-3730-467d-b422-60771eb780de)

**Reply by tjy-gitnub** [reply]
所以快捷键实现了吗？
还是只能在开始菜单里点？Ou0)?o

**Reply by User782Tec** [reply]
> 所以快捷键实现了吗？ 还是只能在开始菜单里点？Ou0)?o

咳咳
不好实现
所以还没实现

**Reply by User782Tec** [reply]
> > ![image](https://user-images.githubusercontent.com/115236204/239657204-ce0efe97-fb57-4b13-b35d-c81e88724f55.png)
> > 运行应用
> 
> 下拉文本框呢？ ![image](https://user-images.githubusercontent.com/121747915/239658575-8ff0f02d-3730-467d-b422-60771eb780de.png)

懒得做了（？

**Comment by User782Tec** [comment]
![image](https://github.com/tjy-gitnub/win12/assets/115236204/7d95c93c-d3f2-44bc-a71d-069dc1054993)

控制中心

**Reply by tjy-gitnub** [reply]
wow 好看
建议把任务栏改成黑色图标，大小一致

**Reply by User782Tec** [reply]
> wow 好看 建议把任务栏改成黑色图标，大小一致

行

**Comment by NB-Group** [comment]
你以后不是要做任务管理器吗，可以参考
![image](https://github.com/tjy-gitnub/win12/assets/121747915/5c14a033-e563-486f-9875-f9d75929e1a9)
来获取当前网页使用的内存但是貌似没办法获取cpu

**Comment by User782Tec** [comment]
![image](https://github.com/tjy-gitnub/win12/assets/115236204/dabd2964-33ba-41d9-9adc-f662e1286f3f)

白板~

**Comment by User782Tec** [comment]
![image](https://github.com/tjy-gitnub/win12/assets/115236204/efd803d4-903a-43ec-b208-60a73e6058b2)

文件资源管理器的面包屑导航

**Comment by User782Tec** [comment]
![image](https://github.com/tjy-gitnub/win12/assets/115236204/f3f2cd02-19a3-4679-90f1-d30d54f31d0b)
本次更新的重头戏~任务管理器！

**Reply by tjy-gitnub** [reply]
🙌哦~~~~🙌

**Comment by User782Tec** [comment]
![image](https://github.com/tjy-gitnub/win12/assets/115236204/c395cecb-6588-47d7-854e-89fa0507c0aa)
系统性能监视器

**Comment by NebulaTechs** [comment]
题外话：真厉害，一个white board占了两GB Memory空间。

**Reply by NebulaTechs** [reply]
哔哩哔哩也挺强的。。。
